examples/atlas-streams/README.md

```diff
@@ -0,0 +1,23 @@
+# Using Atlas Streams with Cloudformatiom
```

Comment:
Atlas stream instance examples will be moved under this directory as well once https://github.com/mongodb/mongodbatlas-cloudformation-resources/pull/930 is merged